CMakeLists.txt

```diff
@@ -77,6 +77,9 @@ option(ENABLE_erd "Enables use of ERD instead of Libint (requires Fortran)" OFF)
 option(ENABLE_simint "Enables use of SIMINT two-electron integral library" OFF)
 option(ENABLE_gdma "Enables Stone's GDMA multipole code (requires Fortran)" OFF)
 option(ENABLE_PCMSolver "Enables PCMSolver library (requires Fortran)" OFF)
+option(ENABLE_ASAN "Enables address sanitizer" OFF)
+option(ENABLE_TSAN "Enables thread santizer" OFF)
+option(ENABLE_UBSAN" Enables undefined behavior sanitizer" OFF)
```

Comment:
What about also adding the MemorySanitizer?

Reply:
Can do.